pegasus/sites.v3/code.org/views/teacher_announcements.haml

```diff
@@ -0,0 +1,16 @@
+%div.teacher-announce
+  %a.announce-heading-link{:href=>"http://teacherblog.code.org/"}
+    %div.announce-heading
+      =I18n.t(:dashboard_announcements)
+      %div.heading-view-all
+        =I18n.t(:dashboard_announce_view_all)
+        >>
+  %div.announce-content
+    %div.content-img-container
+      %img.content-img{:src=>"/images/kids4cropped_2x.jpg"}
```

Comment:
The image is very big, which is okay for a source image because we can ask Pegasus to scale the image down before it serves it (and this is then cached by varnish on the server so it's not rescaled every request).  But we need to make sure we do this server-side scaling.  I've forwarded you details on using the fit/fill/resize URL for an image.